interface/wx/clipbrd.h

```diff
@@ -97,7 +97,7 @@ class wxClipboard : public wxObject
         exit.
 
         Currently this method is not implemented in X11-based ports, i.e.
-        wxGTK, wxX11 and wxMotif and always returns @false there.
+        wxX11 and wxMotif and always returns @false there.
```

Comment:
It would be better to say that this is implemented in MSW and GTK ports and not implemented anywhere else currently.